README.md

```diff
@@ -27,6 +98,104 @@ def App():
 
 run(App)
 ```
+HTML with idiom example
+
+```python
+from idom import html
+
+html.div(
+    html.h1("My Todo List"),
+    html.ul(
+        html.li("Design a cool new app"),
+        html.li("Build it"),
+        html.li("Share it with the world!"),
+    )
+)
+```
+Your first components
+
+```python
+from idom import component, html, run
+
+
+@component
+def Photo():
+    return html.img(
+        {
+            "src": "https://picsum.photos/id/237/500/300",
+            "style": {"width": "50%"},
+            "alt": "Puppy",
+        }
+    )
+
+
+run(Photo)
+
+```
+
+Rendering data
+```python
+from idom import component, html, run
+
+
+@component
+def DataList(items, filter_by_priority=None, sort_by_priority=False):
+    if filter_by_priority is not None:
+        items = [i for i in items if i["priority"] <= filter_by_priority]
+    if sort_by_priority:
+        items = list(sorted(items, key=lambda i: i["priority"]))
+    list_item_elements = [html.li(i["text"], key=i["id"]) for i in items]
+    return html.ul(list_item_elements)
+
+
+@component
+def TodoList():
+    tasks = [
+        {"id": 0, "text": "Make breakfast", "priority": 0},
+        {"id": 1, "text": "Feed the dog", "priority": 0},
+        {"id": 2, "text": "Do laundry", "priority": 2},
+        {"id": 3, "text": "Go on a run", "priority": 1},
+        {"id": 4, "text": "Clean the house", "priority": 2},
+        {"id": 5, "text": "Go to the grocery store", "priority": 2},
+        {"id": 6, "text": "Do some coding", "priority": 1},
+        {"id": 7, "text": "Read a book", "priority": 1},
+    ]
+    return html.section(
+        html.h1("My Todo List"),
+        DataList(tasks, filter_by_priority=1, sort_by_priority=True),
+    )
+
+
+run(TodoList)
+```
+<!---Documatic-section-group: concept-end--->
+
+<!---Documatic-section-group: helloworld-end--->
+
+<!---Documatic-section-group: dev-start--->
+
+
+## Developers
+<!---Documatic-section-dev: setup-start--->
+* This project uses `pre-commit` to enforce code style on commit. Run `pre-commit install` in a terminal to setup
+* Tests are present in `tests/` (using pytest)
```

Comment:
There's a lot more information that could go here - like the fact that you need Node and `npm` installed in your dev environment.